src/main/resources/db/changelog/changelog-1.1.1.yaml

```diff
@@ -0,0 +1,29 @@
+databaseChangeLog:
+  - changeSet:
+      id: changeset-1.1.0_002
+      author: Mikita Shapel
+      changes:
+        - modifyDataType:
+            tableName: student
+            columnName: id
+            newDataType: uuid
+        - modifyDataType:
+            tableName: course
+            columnName: id
+            newDataType: uuid
+        - modifyDataType:
+            tableName: course_settings
+            columnName: id
+            newDataType: uuid
+        - modifyDataType:
+            tableName: lesson
+            columnName: id
+            newDataType: uuid
+        - modifyDataType:
+            tableName: student_course
+            columnName: student_id
+            newDataType: uuid
+        - modifyDataType:
+            tableName: student_course
+            columnName: course_id
+            newDataType: uuid
```

Comment:
this changeset should be in previous changelog, there is no need to create new one